network/src/connectivity_manager/test.rs

```diff
@@ -45,17 +45,21 @@ fn setup_conn_mgr(
         })
         .collect::<HashMap<_, _>>();
 
+    let conn_mgr_config = ConnectivityManagerConfig::new(
+        network_context,
+        Arc::new(RwLock::new(eligible_peers)),
+        seed_peers,
+        ticker_rx,
+        FixedInterval::from_millis(100),
+        300, /* ms */
+    );
+
     let conn_mgr = {
```

Comment:
what is the purpose of these extra `{` and `}`

Reply:
there is none.